tests/test_blotter.py

```diff
@@ -83,64 +89,59 @@ def CREATE_BARDATA_DATA_FREQUENCY(cls):
                            (StopLimitOrder(10, 20), 10, 20)])
     def test_blotter_order_types(self, style_obj, expected_lmt, expected_stp):
 
-        blotter = Blotter('daily', self.env.asset_finder)
+        blotter = Blotter('daily', self.asset_finder)
 
-        asset_24 = blotter.asset_finder.retrieve_asset(24)
-        blotter.order(asset_24, 100, style_obj)
-        result = blotter.open_orders[asset_24][0]
+        blotter.order(self.asset_24, 100, style_obj)
+        result = blotter.open_orders[self.asset_24][0]
 
         self.assertEqual(result.limit, expected_lmt)
         self.assertEqual(result.stop, expected_stp)
 
     def test_cancel(self):
-        blotter = Blotter('daily', self.env.asset_finder)
+        blotter = Blotter('daily', self.asset_finder)
 
-        asset_24 = blotter.asset_finder.retrieve_asset(24)
```

Comment:
Not super important, but we might make this diff less noisy by just replacing this line with `asset_24 = self.asset_24` instead of changing every usage below.